v24/src/main/java/com/foursoft/harness/kbl/v24/HasProcessingInformation.java

```diff
@@ -25,10 +25,58 @@
  */
 package com.foursoft.harness.kbl.v24;
 
+import com.foursoft.harness.kbl.v24.util.StreamUtils;
+
 import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
 
 public interface HasProcessingInformation {
 
     List<KblProcessingInstruction> getProcessingInformations();
 
+    /**
+     * Filters the list of {@link KblProcessingInstruction} key.
```

Comment:
Please add to those java docs that at most one element of the given typ is expected. If more a found, the first one will be returned and a warning is logged. 
If multiple values of the given type are valid, than please use the other #getProcessingInstructionValues(...) method.